config/jobs/kubernetes/sig-cloud-provider/azure/release-1.21.yaml

```diff
@@ -341,7 +341,7 @@ periodics:
         make e2e-test
       env:
       - name: KUBERNETES_VERSION
-        value: "latest"
+        value: "latest-1.21"
```

Comment:
@andyzhangx these files are all generated, they shouldn't be modified manually otherwise changes will be overwritten next time there are generated again. You need to update https://github.com/andyzhangx/test-infra/blob/baaf24408f921fbafd76bc109148591964006d14/config/jobs/kubernetes/sig-cloud-provider/azure/generate.sh

Also, I think we need to update 1.23 and 1.24 too, they are also using "latest" https://github.com/andyzhangx/test-infra/commit/c085888bd2041c1563766afb652cd32f8184dcda#diff-50eaee83e79264b43f2eb13dd562ae7eb39d7beb0c37899e39b76309607582fe

Reply:
thanks, there is a follow up PR here: https://github.com/kubernetes/test-infra/pull/26580, PTAL.